gfx-memory/src/allocator/linear.rs

```diff
@@ -150,42 +177,43 @@ impl<B: Backend> LinearAllocator<B> {
         self.line_size / 2
     }
 
-    fn cleanup(&mut self, device: &B::Device, off: usize) -> Size {
+    fn cleanup(&mut self, device: &B::Device, free_memory: bool) -> Size {
         let mut freed = 0;
-        while self.lines.len() > off {
+        for _ in 0..self.lines.len() {
```

Comment:
nit: could be `while !self.lines.is_empty()`

Reply:
Sure